lib/Controller/Display.php

```diff
@@ -794,6 +796,23 @@ function editForm($displayId)
             $layouts = [];
         }
 
+        $playerVersions  = $this->playerVersionFactory->query(null, ['playerType' => $display->clientType]);
+        $versions = [];
+
+        // Go through the player versions from factory and supply it as a parameter versions to edit form - on the form we default to value set previously on the profile or to the newest Code version if it's a new display profile.
+        foreach ($playerVersions as $playerVersion) {
```

Comment:
opportunity to refactor this into a method on the display? `$display->getSetting('versionMediaId', overrideOnly:true)`